src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -290,7 +310,7 @@ Type[] IInvocationBinder.GetParameterTypes(string methodName)
             Type[] types;
             if (!_paramTypes.TryGetValue(methodName, out types))
             {
-                throw new InvalidOperationException($"The hub method '{methodName}' could not be resolved.");
+                return Type.EmptyTypes;
```

Comment:
Ending the connection for an unknown method is in all cases in incorrect. We need to expand the interface so that we know we *have* to end the connection if the format isn't self describing.